src/java/org/apache/cassandra/index/sai/disk/v1/IndexWriterConfig.java

```diff
@@ -57,6 +60,10 @@ public class IndexWriterConfig
 
     public static final int DEFAULT_MAXIMUM_NODE_CONNECTIONS = 16;
     public static final int DEFAULT_CONSTRUCTION_BEAM_WIDTH = 100;
+    // Use non-positive value to disable it. When num of vectors in vector memtable index reaches the threshold, it triggers flush
+    public static final int DEFAULT_VECTOR_FLUSH_TRESHOLD = Integer.getInteger("dse.default_vector_flush_threshold", -1);
```

Comment:
Nit: Consider making this easier to specify, allowing things like "1B" or "10M". 

Note: `FBUtilities.parseHumanReadable(value, null, null)` would work but may be confusing as it accepts SI units (e.g. "1G" or "10k") and would turn "10m" into "0.01".

Reply:
Don't use `dse.` for the prefix, there is no dse, just use `cassandra.`.  Also this should use CassandraRelevantProperties?

Reply:
> Don't use dse. for the prefix, there is no dse, just use cassandra.

right. muscle memory..